src/server/templates/page.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function page({ body = "", assets, title }) {
```

Comment:
Any reason not to favor arrow syntax here (you wouldn't even need return or brackets)? Since this is our very first function that we've defined, would you mind adding a JSDoc with the typing? This will also give us a chance to test the JSDoc linting plugin you've added! Something like:

```js
/**
 * @arg {?string} body HTML
 * @arg {{entry: ?string, asset: ?object}} assets Manifest of filename entry points to bundled assets.
 * @arg {?string} title
 * @return {!string} HTML
 */```